### ISSUE TITLE
ReferenceError: Symbol is not defined with bootstrap-rubygem

### ISSUE THREAD

**Issue by drewscott** [ai/autoprefixer-rails#116]
It seems with the new release (7.0) is breaking with twbs/bootstrap-rubygem ('bootstrap', '~> 4.0.0.alpha6')

`ActionView::Template::Error (ReferenceError: Symbol is not defined)`

I got around this issue by putting `gem 'autoprefixer-rails', '~> 6.7'` in my Gemfile

**Comment by ai** [comment]
Can you give me a full stacktrace?

**Comment by glebm** [comment]
Also happens with bootstrap-sass but I don't think it's related to these gems (I am the author of both).

The backtrace is thus:

```
ExecJS::ProgramError: ReferenceError: Symbol is not defined
Object.parse ((execjs):934:125)
Supports.process ((execjs):6700:28)
(execjs):5642:37
(execjs):12958:28
(execjs):12810:26
Root.each ((execjs):12776:22)
Root.walk ((execjs):12809:21)
Root.walkAtRules ((execjs):12956:25)
Processor.add ((execjs):5631:13)
plugin ((execjs):884:32)
.bundle/ruby/2.4.0/gems/execjs-2.7.0/lib/execjs/external_runtime.rb:39:in `exec'
.bundle/ruby/2.4.0/gems/execjs-2.7.0/lib/execjs/external_runtime.rb:21:in `eval'
.bundle/ruby/2.4.0/gems/execjs-2.7.0/lib/execjs/external_runtime.rb:46:in `call'
.bundle/ruby/2.4.0/gems/autoprefixer-rails-7.0.0/lib/autoprefixer-rails/processor.rb:36:in `process'
```

I think the issue is that this version of `autoprefixer.js` uses `Symbol.iterator` which is not supported by most ExecJS runtimes.

**Comment by ai** [comment]
Yeap, it related to ExecJS runtime. I will try to add some Symbol polyfill to JS runtime.

As quick solution — use latest Node.js for ExecJS runtime.

**Comment by glebm** [comment]
Thanks, worked with node v4.2.6

**Comment by ai** [comment]
@glebm @drewscott what ExecJS env did you use? (I need it to reproduce error)

**Comment by glebm** [comment]
I had used node v0.10.6, but this should reproduce with e.g. MiniRacer and most other non-node runtimes.

**Comment by ai** [comment]
I think to add warning on this case. Because PostCSS and Autoprefixer itself doesn’t support node.js <= 0.12 itself.

At least it works on `v8` and Node.js >= 4.

Is this plan is OK for you?

**Comment by glebm** [comment]
As long as it works with MiniRacer (for Heroku compatibility), sounds good to me.

**Comment by ai** [comment]
@glebm can you help and add `MiniRacer` to Travis CI? I am not sure how to test it, but I agree that Heroku compatibility is important.

**Comment by drewscott** [comment]
@ai Initially the error showed for therubyracer v0.12.3 then I tried node v0.10.29 with the same error. Thanks guys for looking into this so fast.

**Comment by glebm** [comment]
@drewscott therubyracer's v8 runtime is too old to support autoprefixer. Replace `therubyracer` in your Gemfile with `mini_racer`, or upgrade to Node v0.12+.

**Comment by drewscott** [comment]
@glem thanks, working well with latest node v6.10.3

**Comment by ai** [comment]
Fixed 9ee53ef

**Comment by ai** [comment]
Released in 7.0.1
